### ISSUE TITLE
Discord Vote Tag is Out of Date

### ISSUE THREAD

**Issue by subbob** [DraconisCluster/DX-Issues#7]
** tl; dr **
The current Vote tag refers to the sigma claim, sigma tokens, etc system.

** Full info **
Update the vote tag and incorporate it's use into welcoming new players. For example, use it when adding the scripter role.

** Notes **

1. I drafted the replacement prior to reading the current one.  May want to use portions of both in the updated vote tag.
2. Although I referenced the format help at [Template Designer Documentation](https://jinja.palletsprojects.com/en/3.1.x/templates/) I am unsure as to there the <a href=""></a> syntax in my proposed draft will work.

**References**
Current Vote Tag
[current-vote.jinja.txt](https://github.com/DraconisCluster/DX-Issues/files/13851623/current-vote.jinja.txt)
Proposed Vote Tag
[proposed-vote.jinja.txt](https://github.com/DraconisCluster/DX-Issues/files/13851627/proposed-vote.jinja.txt)

**Comment by Terif1** [comment]
While using it as part of say scripter is at the very least something we'd run through support first, I would say having a new expanse specific tag would be good to do.

**Comment by Synirrr** [comment]
Has any work occurred on this?

**Comment by draconb** [comment]
Was this tag completely removed? I don't see it anymore when checking for similar to "vote" or "reward"
